code/style.css

```diff
@@ -1,7 +1,339 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  margin: 0;
+  background: #E1D2CB;
+  font-family: 'Montserrat', sans-serif;
+}
+
+
+/* -------------------NAVIGATIONBAR--------------------------- */
+
+.navbar {
+  width: 100%;
+  font-size: 15px;
+  font-weight: bold;
+  background-color: whitesmoke;
+  opacity: 0.5;
+  position: relative; 
+  z-index: 15;
+  display: flex; 
+  object-fit: cover;
+  overflow: hidden;
+  justify-content: space-evenly;
+  align-items: center;
+  float: right;
+  padding: 20px 0px 20px 0px;
+}
+
+
+/* ---------------------HERO VIDEO AND CAPTION----------------- */
+
+.hero {
+  height: 100vh;
+  width: 100%;
+  background: white;
+
+}
+.caption {
+  width: 100%;
+  height: 100%;
+  z-index: 20;
+  position: relative;
+  text-align: center; 
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.caption h1 {
+  background-color: #DECDB4;
+  opacity: 0.6;
+  padding: 5px 10px 5px 10px;
+  border-radius: 5px;
+}
+
+.caption h2 {
+  background-color: #DECDB4;
+  opacity: 0.6;
+  padding: 5px 10px 5px 10px;
+  border-radius: 5px;
+}
+
+
+.hero-button {
+  background-color:whitesmoke; 
+  opacity: 0.5;
+  border: 1px solid whitesmoke;
+  width:180px;
+  padding: 10px;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+a:hover {
+  text-decoration: underline;
+  font-size: 20px;
+}
+
+
+video { 
+  width: 100%; 
+  height: 100vh;
+  position: absolute; 
+  top: 0;
+  left: 0;
+  z-index: 5; 
+  object-fit: cover;
+  overflow: hidden;
+}
+
+
+/* -------------------SIGN-UP-FORM----------------------- */
+
+.flex-container {
+  display: flex; 
+  flex-direction: column;
+  row-gap: 1em;
+  justify-content: center;
+  align-items: center;
+  margin-top: 3em;
+}
+
+
+.sign-up-form {
+  border: 2px solid #DECDB4;
+  border-radius: 10px;
+  background-color: #F3E9E0;
+  font-family: 'Montserrat', sans-serif; 
+  width: 350px; 
+  height: auto;
+  padding: 30px 0px 30px 0px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
```

Comment:
Im unsure what align items does at this part, it doesnt show any changes for me on the inspector tool.